src/main/java/org/isf/medicalinventory/service/MedicalInventoryIoOperationRepository.java

```diff
@@ -49,12 +49,12 @@ public interface MedicalInventoryIoOperationRepository extends JpaRepository<Med
 	List<MedicalInventory> findInventoryByParams(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("status") String status, @Param("type") String type);
 	
 	@Query(value = "select medinv from MedicalInventory medinv where medinv.inventoryDate >= :dateFrom and medinv.inventoryDate < :dateTo and medinv.inventoryType = :type")
-	List<MedicalInventory> findInventoryByParamsPageablewithoutSatus(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("type") String type);
+	List<MedicalInventory> findInventoryByParamsWithoutStatus(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("type") String type);
 	
 	@Query(value = "select medinv from MedicalInventory medinv where medinv.inventoryDate >= :dateFrom and medinv.inventoryDate < :dateTo and medinv.status = :status and medinv.inventoryType = :type")
 	Page<MedicalInventory> findInventoryByParamsPageable(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("status") String status, @Param("type") String type, Pageable pageable);
 	
 	@Query(value = "select medinv from MedicalInventory medinv where medinv.inventoryDate >= :dateFrom and medinv.inventoryDate < :dateTo and medinv.inventoryType = :type")
-	Page<MedicalInventory> findInventoryByParamsPageablewithoutSatus(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("type") String type, Pageable pageable);
+	Page<MedicalInventory> findInventoryByParamsWithoutStatusPageable(@Param("dateFrom") LocalDateTime dateFrom, @Param("dateTo") LocalDateTime dateTo, @Param("type") String type, Pageable pageable);
```

Comment:
Anyway, why do we need to specify `WithoutSomething`? Would be better to just use a positive form like `BetweenDate` instead of a generic `ByParams`?

Reply:
okay let me update it